index.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+Generate random numbers that are consecutively unique.
+
+@returns A function that when called will return a random number that is never the same as the previous.
+
+@example
+```
+import uniqueRandom = require('unique-random');
+const random = uniqueRandom(1, 10);
+
+console.log(random(), random(), random());
+//=> 5 2 6
+```
+*/
+declare function uniqueRandom(
+	minimumValue: number,
+	maximumValue: number
```

Comment:
I'm starting to question whether this is actually an improvement. The `Value` part doesn't really add anything, as it's implied that it's a value. And using `maximum` over `max` is fine I guess, but `max` is even more common than `maximum` and is considered a standalone word, not just an abbreviation, but `min` is an abbreviation, so for consistency, I think `maximum` and `minimum` would be the best names. What do you think?

Reply:
I find your reasoning very conclusive, I was also very unsure about the renaming. Let's go with `minimum` and `maximum`.